src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -2070,4 +2070,12 @@ public String exportFilter(UUID studyUuid, UUID filterUuid) {
         // will use root node network of the study
         return filterService.exportFilter(networkStoreService.getNetworkUuid(studyUuid), filterUuid);
     }
+
+    public String getVoltageInitResult(UUID nodeUuid) {
+        return voltageInitService.getVoltageInitResultOrStatus(nodeUuid, "");
+    }
+
+    public String getVoltageInitStatus(UUID nodeUuid) {
+        return voltageInitService.getVoltageInitResultOrStatus(nodeUuid, "/status");
+    }
```

Comment:
I moved these methods when i needed to use the parameters uuid to get the result in my previous code ... 
But, you're ok, now it can be moved again to voltageInitService ...